.github/ISSUE_TEMPLATE/convergence_epic.md

```diff
@@ -1,48 +1,55 @@
+  - [link to PR]
+  - [link to package: `https://github.com/microsoft/fluentui/tree/master/packages/react-components/react-<your-component>`]
+- [ ] (Optional) [Draft implementation](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#draft-implementation)
+  - [link to PR, if applicable]
+- [ ] [Component Spec authored](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#component-spec) and [reviewed](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#spec-review)
+  - [link to PR]
+  - [link to Spec.md: `https://github.com/microsoft/fluentui/tree/master/packages/react-components/react-<your-component>/Spec.md`]
 
 ## Implementation
 
-[link to react-* package folder]
-
-- [ ] Component implementation
+- [ ] [Component implementation](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#implementation)
   - [link(s) to component implementation related PRs]
-- [ ] Storybook stories
-  - [link(s) to stories PRs]
-- [ ] Add tests: Conformance, Unit, and VR -- add PRs to all
-  - [ ] Bundle size fixtures
-  - [ ] Conformance tests
-  - [ ] Unit tests
-  - [ ] VR tests
-  - [ ] Accessibility behavior tests
-  - [ ] Create an issue and run [manual accessibility tests](https://github.com/microsoft/fluentui/wiki/Manual-Accessibility-Review-Checklist): (issue link)
-  - [ ] Performance test scenario
-- [ ] README.md covering basic usage
-- [ ] MIGRATION.md guide (include v8 and v0)
+- [ ] Initial conformance and unit tests (validate basic functionality)
+- [ ] [Initial documentation](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#documentation)
+  - [ ] [Storybook stories](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#storybook-stories)
+  - [ ] README.md covering basic usage
+  - [ ] MIGRATION.md guide (include v8 and v0)
+- [ ] [Component released as unstable](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#unstable-release) from `@fluentui/react-components/unstable`
+  - [link to PR]
 
 ## Validation
 
-- [ ] Add and validate in UI Builder
-- [ ] Add and validate in docs site
-- [ ] Validate with token pipeline
-- [ ] Validate in product
-- [ ] Finalize migration guide
-  - [ ] Author codemods
+- [ ] [Add tests](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#tests)
+  - [ ] Unit and conformance tests
+  - [ ] VR tests
+  - [ ] Bundle size fixtures
+  - [ ] Performance test scenario
+  - [ ] Accessibility behavior tests
+  - [ ] Create an issue and run [manual accessibility tests](https://github.com/microsoft/fluentui/wiki/Manual-Accessibility-Review-Checklist): [link to issue]
+  - [Link to PR(s)]
+- [ ] [Validate in product](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#validation)
```

Comment:
I'd recommend this be "validate with partners". Basically this is where we get Partner sign off on a component before going stable (if possible)
```suggestion
- [ ] [Validate with partners](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#validation)
```